axlearn/experiments/text/gpt/fuji.py

```diff
@@ -188,6 +189,10 @@ def get_trainer_kwargs(
                 num_kv_heads=None if version == Version.V1 else 8,
                 rope_theta=rope_theta,
                 flash_attention=flash_attention,
+                remat_spec=RematSpec(
+                    prevent_cse=True,
```

Comment:
not sure if we actually need prevent_cse=True. It may work with false too.

Reply:
we may also want this to be conditional based on instance type. E.g. if v5e then use remat_spec.policy = None otherwise use default remat_spec

Reply:
It doesn't seem like there is an easy way to add such a conditional though